eclass/git-r3.eclass

```diff
@@ -1054,6 +1137,13 @@ git-r3_src_unpack() {
 	_git-r3_env_setup
 	git-r3_src_fetch
 	git-r3_checkout
+
+	if [[ ! ${EGIT_LFS} && ${_EGIT_LFS_FILTERS_FOUND} ]]; then
+		eqawarn "QA Notice: There are Git LFS filters setup in the cloned repo, consider using EGIT_LFS!"
+	fi
+	if [[ ${EGIT_LFS} && ! ${_EGIT_LFS_FILTERS_FOUND} ]]; then
+		eqawarn "QA Notice: There are no Git LFS filters setup in the cloned repo. EGIT_LFS will do nothing!"
+	fi
```

Comment:
I guess you're trying to avoid reporting it multiple times *but* by design the eclass doesn't require `src_unpack` being called — you can just call `_src_fetch` and `_checkout` directly.

Reply:
Yepp. I didn't want to spam it in case the repo had multiple submodules.

Do you think spamming is fine?
In that case I can remove the global variable and change the message to something like:
"${repo} has Git LFS filters installed! Consider doing X"

Reply:
Hmm, submodules you should be able to handle because these are handled recursively. What would be really problematic is that you'd get false positives in ebuilds calling `checkout` more than once. So maybe let's leave it as-is.